admin/package.json

```diff
@@ -78,6 +78,9 @@
     "xregexp": "^4.2.0"
   },
   "devDependencies": {
+    "@babel/core": "^7.6.4",
+    "@babel/node": "^7.6.3",
+    "@babel/preset-env": "^7.6.3",
```

Comment:
I may be missing something here.  Where are you specifying the target output for babel?